_plugins/build.rb

```diff
@@ -0,0 +1,8 @@
+`npm install`
+
+production = ENV['BRANCH'] == 'master' or ENV['BRANCH'] == 'staging' or ENV['BRANCH'] == 'js-optimization'
+
+# ENV['NODE_ENV'] = if production then 'prod' else 'dev' end
+ENV['NODE_ENV'] = 'prod'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.